src/components/layout/drawer/Actions.jsx

```diff
@@ -157,20 +157,22 @@ export default function DrawerActions() {
           <ListItemText primary={t(loggedIn ? 'logout' : 'login')} />
         </ListItem>
       )}
-      <a
-        href="https://github.com/WatWowMap/ReactMap"
-        referrerPolicy="no-referrer"
-        target="_blank"
-        rel="noreferrer"
-        style={{ textDecoration: 'none', color: 'white' }}
-      >
-        <ListItem button>
-          <ListItemIcon>
-            <HeartIcon color="primary" />
-          </ListItemIcon>
-          <ListItemText primary={t('contribute')} />
-        </ListItem>
-      </a>
+      {config.map.enableContribute && (
```

Comment:
I've been _very_ lenient on what things are configurable in this project, including, removing any references to this project, but I'm going to put my foot down on this one. There are plenty of users that aren't in the mapping sphere that are able to contribute, and with all of the time and effort I've put into it, I think it's perfectly reasonable to leave a single link for those that are interested to find the source. Even if it only attracts a single person who wants to contribute and make the project better, then it's worth it.

I would be willing to bet that this person who recently starred the repo found it from this link :)
<img width="395" alt="Screenshot 2023-07-18 at 2 45 45 PM" src="https://github.com/WatWowMap/ReactMap/assets/58572875/1b602efe-4a21-4e22-8c28-5a6e7fd0c205">

Reply:
I agree that the option to contribute should be available, and I agree with your arguments as well @TurtIeSocks. Perhaps under Menu - Settings is the best place, perhaps not. I'm not sure where else to put it though as it shouldn't be hidden, My small concern is that my community believes they are contributing to me - I have had a few questions about it already. I explained what you are saying as well, and that all suggestions and issues are welcome to put there. Most of the stuff however is resolved within the community because an issue is rarely an issue but instead a user error (or not well enough explained. which could be a contribution). Not sure if it makes any sense what I'm saying 😄